evalml/pipelines/components/transformers/imputers/time_series_imputer.py

```diff
@@ -152,8 +155,10 @@ def transform(self, X, y=None):
         X_not_all_null = X.ww.drop(self._all_null_cols)
         X_schema = X_not_all_null.ww.schema
         X_schema = X_schema.get_subset_schema(
+            # --> note that I'm including it right now so I can test fully even though this will go once we integrate into automlsearch
+            # https://github.com/alteryx/evalml/issues/4001#issuecomment-1443894376
             subset_cols=X_schema._filter_cols(
-                exclude=["IntegerNullable", "BooleanNullable"],
+                exclude=["IntegerNullable", "BooleanNullable", "AgeNullable"],
```

Comment:
We'll likely remove this logic once we integrate the new nullable type handling into AutoMLSearch, but I wanted to add in `AgeNullable` here so that we could test with it now.

The one reason we may keep is is that nans are present at this stage, so we'll have to convert types to `Double` and `Categorical`, and we may want logic to not keep those types once nans are gone and we can use `Integer` and `Boolean`. I'll put more thought into this when I get to https://github.com/alteryx/evalml/issues/3999